meson.build

```diff
@@ -23,6 +23,9 @@ libavutil = dependency('libavutil', version: '>= 56.70.100')
 libswresample = dependency('libswresample', version: '>= 3.9.100')
 libswscale = dependency('libswscale', version: '>= 5.9.100')
 
+libplacebo = dependency('libplacebo', version: '>=6.292.0', fallback: ['libplacebo', 'libplacebo'],
+    default_options: ['default_library=static', 'demos=false'])
```

Comment:
This can be reverted back to simple 

`libplacebo = dependency('libplacebo', version: '>=6.292.0')`